pandas/core/groupby/generic.py

```diff
@@ -1710,3 +1714,69 @@ def _normalize_keyword_aggregation(kwargs):
         order.append((column,
                       com.get_callable_name(aggfunc) or aggfunc))
     return aggspec, columns, order
+
+
+def _make_lambda(func, i):
+    def f(*args, **kwargs):
+        return func(*args, **kwargs)
+    f.__name__ = "<lambda_{}>".format(i)
+    return f
+
+
+def _managle_lambda_list(aggfuncs):
+    i = 0
+    aggfuncs2 = []
+    for aggfunc in aggfuncs:
+        if com.get_callable_name(aggfunc) == "<lambda>":
+            if i > 0:
+                aggfunc = _make_lambda(aggfunc, i)
+            i += 1
+        aggfuncs2.append(aggfunc)
+
+    return aggfuncs2
+
+
+def _maybe_mangle_lambdas(agg_spec):
+    """
```

Comment:
can add types

Reply:
you add the type here (I know its trivial, but Any is instructive here)

Reply:
Is it instructive? It's not exactly correct, so I think it's worse than not having one.

Reply:
Any is never technically *incorrect*; behavior is practically the same as having no annotation. Indifferent on whether required here just wanted to clarify that @TomAugspurger

Reply:
the reason to put it, is that it IS indicative. Not have a type in general (when everything is typed) is not great.